ai_diffusion/ui/server.py

```diff
@@ -432,6 +441,14 @@ def __init__(self, srv: Server, parent=None):
                 is_expanded=False,
                 parent=self,
             ),
+            "qwen": PackageGroupWidget(
```

Comment:
This will be confusing as long as it's just an empty group, I think the managed install can also be added later.

Reply:
Must be one of those unnecessary code lines I added when copying flux arch at the start.